src/types/method.ts

```diff
@@ -0,0 +1,29 @@
+export type Method = {
+  name: string;
+  namespace: MethodNamespace;
+  description: string;
+  parameters?: MethodVariable[];
+  return: MethodVariable;
+  example?: MethodExample;
+};
+
+export type MethodVariable = {
+  type: MethodVariableType;
+  description: string;
+};
+
+export enum MethodVariableType {
+  String,
+  Data,
```

Comment:
What are these two types, e.g. for passing a block number which type would that be? I think some RPC methods might also have array inputs but can't recall offhand

Reply:
I think it would be `QUANTITY`, as referenced here: https://ethereum.org/en/developers/docs/apis/json-rpc#eth_blocknumber. I will expand this type as I add new RPC methods.